src/App.js

```diff
@@ -20,10 +20,8 @@ export default function App() {
             <main>
                 <AddTodo
                     setTodoItems={setTodoItems}
-                    editId={editId}
-                    setEditId={setEditId}
-                    itemToEdit={editId ? todoItems.find((e) => e.id === editId) :undefined}
-
+                    setItemToEdit={setItemToEdit}
+                    itemToEdit={itemToEdit ? todoItems.find((e) => e.id === itemToEdit) :undefined}
```

Comment:
Once you implement the comment for line 14, you can now change this to `itemToEdit={itemToEdit}`